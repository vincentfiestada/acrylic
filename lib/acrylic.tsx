/**
 * Re-export all components in one place
 */
export { default as IProps } from "./components/IProps";
export { default as IEventProps } from "./components/IEventProps";

/**
 * Layout Components
 */
export { default as ILayoutProps } from "./components/layout/ILayoutProps";
export { default as Section } from "./components/layout/Section";
export { default as Column } from "./components/layout/Column";
export { default as Row } from "./components/layout/Row";
export { default as Nav } from "./components/layout/Nav";
export { default as Breadcrumb, Crumb } from "./components/layout/Breadcrumb";
export { default as Alert } from "./components/layout/Alert";
export { default as Toolbar } from "./components/layout/Toolbar";
export { default as Caption } from "./components/layout/Caption";
export { default as Divider } from "./components/layout/Divider";
export { default as Folder } from "./components/layout/Folder";
export {
	default as Table,
	TableSection,
	TableRow,
	Cell
} from "./components/layout/Table";
export { default as Card, CardBody } from "./components/layout/Card";
export {
	default as Menu,
	MenuItem,
	SubMenu,
	MenuHeading
} from "./components/layout/Menu";

/**
 * Media Components
 */
export { default as Image } from "./components/media/Image";
export { default as Icon } from "./components/media/Icon";

/**
 * Typography Components
 */
export { default as ITextProps } from "./components/typography/ITextProps";
export { default as Title } from "./components/typography/Title";
export { default as Text } from "./components/typography/Text";

/**
 * Control Components
 */
export { default as Button } from "./components/controls/Button";
export { default as IconButton } from "./components/controls/IconButton";
export { default as Dropdown } from "./components/controls/Dropdown";

/**
 * Variants
 */
export { default as Variant } from "./components/Variant";
