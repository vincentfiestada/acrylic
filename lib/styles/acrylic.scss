@import "normalize.css";
@import "typography";
@import "controls";
