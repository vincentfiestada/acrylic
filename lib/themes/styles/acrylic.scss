@import "normalize.css";
@import "typography";
@import "icons";
@import "layout";
@import "controls";
@import "progress";
@import "animation";
