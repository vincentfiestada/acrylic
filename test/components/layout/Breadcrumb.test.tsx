import * as React from "react";
import { mount } from "enzyme";
import { Breadcrumb, Crumb } from "../../../lib/acrylic";

describe("<Breadcrumb />", () => {
	it("Should match the latest snapshots", () => {
		expect(
			mount(
				<Breadcrumb>
					<Crumb>Item</Crumb>
					<Crumb>Item</Crumb>
					<Crumb>Item</Crumb>
				</Breadcrumb>
			)
		).toMatchSnapshot();
	});
});
