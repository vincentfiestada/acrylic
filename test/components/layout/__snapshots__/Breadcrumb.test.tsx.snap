// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> Should match the latest snapshots 1`] = `
<Breadcrumb>
  <nav
    className="acr-breadcrumb"
  >
    <ul>
      <Crumb>
        <li
          className="acr-crumb acr-li-text-only"
        >
          Item
        </li>
      </Crumb>
      <Crumb>
        <li
          className="acr-crumb acr-li-text-only"
        >
          Item
        </li>
      </Crumb>
      <Crumb>
        <li
          className="acr-crumb acr-li-text-only"
        >
          Item
        </li>
      </Crumb>
    </ul>
  </nav>
</Breadcrumb>
`;
